IntAct.pm

```diff
@@ -164,6 +152,15 @@ sub new {
     $output_rest = 1;
   }
 
+  my $species = $self->config->{species};
+  $self->{species} = $species;
+
+  # get species taxonomy id using ensembl REST API
+  my $species_tax_id = $self->_get_species_tax_id($species);
+
+  die "ERROR: could not get species taxonomy id and the species is not human\n" 
+    if not defined $species_tax_id and $species ne "homo_sapiens";
```

Comment:
If we don't support non-human in this version, does not make sense try to get tax_id if non-human.

```suggestion
  my $species = $self->config->{species};
  $self->{species} = $species;

  die "ERROR: species is not human\n" 
    if $species ne "homo_sapiens";

  # get species taxonomy id using ensembl REST API
  my $species_tax_id = $self->_get_species_tax_id($species);
```

Reply:
I understood that, VEP CLI will have support for non-humans and web VEP will not. That is why we are keeping the PR as is.

We only have homo_sapiens in config file so web VEP will only have human  and not other species
https://github.com/Ensembl/VEP_plugins/blob/7197651f3498dc9224655fd7a821e9dcd6f47bad/plugin_config.txt#L1638

Reply:
But if someone is trying to use CLI will not get this error for non-human?

Reply:
@nakib103 updated me on slack. Closing this question.